src/modules/create_pdf.py

```diff
@@ -59,7 +59,11 @@ def _process(directory: str, name: str, queue: Queue) -> None:
         files: list[str] = [os.path.join(directory, item) for item in os.listdir(directory)]
         if CreatePDF.check_already_has_pdf(files):
             return
-        pdf_file: bytes | None = img2pdf.convert(files)
+        try:
+            pdf_file: bytes | None = img2pdf.convert(files)
+        except img2pdf.ImageOpenError as _e:
+            logger.error(_e)
+            return
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider enhancing error logging for better troubleshooting.

Including the directory name in the error message could help in identifying which directory's processing failed.

```suggestion
            logger.error(f"Error opening image for PDF conversion in directory '{os.path.dirname(files[0])}': {_e}")
            return
```